e2e/cypress/integration/system_console/team_and_channel_assign_roles_spec.js

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('System Console', () => {
+    it('MM-20059 - System Admin can map roles to groups from Team Configuration screen', () => {
+        cy.apiLogin('sysadmin');
+
+        // # Go to system admin page and to team configuration page of channel "eligendi"
+        cy.visit('/admin_console/user_management/teams');
+        cy.findByTestId('eligendiedit').click();
+
+        // # Wait until the groups retrieved and show up
+        cy.wait(500); //eslint-disable-line cypress/no-unnecessary-waiting
+
+        // # Remove all exisiting groups
+        cy.get('#groups-list--body').then((el) => {
+            if (el[0].childNodes[0].innerText !== 'No groups specified yet') {
+                for (let i = 0; i < el[0].childNodes.length; i++) {
+                    cy.get('#group-actions').click();
+                }
+
+                // # Save the setting
+                cy.get('#saveSetting').click();
+            }
+        });
+
+        // # Add the first group in the group list then save
+        cy.findByTestId('add-group').click();
+        cy.get('#multiSelectList').first().click();
+        cy.get('#saveItems').click();
+        cy.get('#saveSetting').click();
```

Comment:
Can you add an assertion to verify that the default role is `Member` when a Group is added.

Reply:
Done! I have made it so that we ensure the default option after we add a channel or team is member.